pages/vi/vi-bellapps.md

```diff
@@ -51,12 +51,16 @@ First, click the "Update Available" button and it will reload your homepage with
 
 We developed a course for aspiring virtual interns so that we could test the software used to build courses and find bugs/things to improve. It is your job to take the course and find out what needs to be fixed/improved from the student standpoint.
 
-To get to Publications, click on "Manager" from the dashboard, then the publications button. Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses out in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated. 
+To get to Publications, click on "Manager" from the dashboard, then the publications button. Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses out in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated.  
+**STEPS:**   
+1.Click on Courses (next to Library).  
+2.There is a course called Virtual Interns.  
+3.You will see a list of steps that test your knowledge of the discussed concepts from virtual interns first-steps.  
```

Comment:
Try to use markdown elements to make these steps a little cleaner.

Also, try to reword step 2 and step 3. Try to make it more concise and to the point.

Reply:
Okay, I will do that and let you know soon.